content/page/dapps/2.html

```diff
@@ -3,6 +3,8 @@
 url: /dapps/2
 aliases:
 - /dapps/2.html
+- /dapps/2_launch.html
```

Comment:
why those new url's?

Reply:
To keep it standardized
 - /dapps/1_launch/
 - /dapps/1/
 - /dapps/2_launch/

It used to be just /dapps/2/ for the launch, now we have the launch included. So when the results come in, we do /dapps/2/

Reply:
ok